ludwig/models/modules/sequence_decoders.py

```diff
@@ -149,104 +149,56 @@ class SequenceTaggerDecoder(Layer):
     def __init__(
             self,
             initializer=None,
-            regularize=True,
+            use_bias=True,
+            kernel_initializer='glorot_uniform',
+            bias_initializer='zeros',
+            kernel_regularizer=None,
+            bias_regularizer=None,
+            activity_regularizer=None,
             attention=False,
             num_classes=0,
-            loss=None,
+            is_timeseries=None,
```

Comment:
this should be a bool, so default is False

Reply:
done